test/e2e/harness/src/app/app.module.ts

```diff
@@ -94,18 +90,13 @@ const appRoutes: Routes = [
   }
 ];
 
-// To perform end-to-end PKCE flow we must be configured on both ends: when the login is initiated, and on the callback
-// The login page is loaded with a query param. This will select a unique callback url
-// On the callback load we detect PKCE by inspecting the pathname
-const url = new URL(window.location.href);
-const pkce = !!url.searchParams.get('pkce') || url.pathname.indexOf('pkce/callback') >= 0;
-const redirectUri = window.location.origin + (pkce ? '/pkce/callback' : '/implicit/callback');
+const redirectUri = window.location.origin + '/login/callback';
 
 const config = {
-  issuer: process.env.ISSUER,
-  pkce,
+  issuer: 'https://dev-4857159.okta.com/oauth2/default', //process.env.ISSUER,
```

Comment:
Looks like you left some dev values here.

Also lets remove the line for `pkce: false`. The test will use pkce (but this isn't relevant to VUE sdk)

Reply:
Done